broker/README.md

```diff
@@ -42,7 +42,8 @@ install KEDA as documented [here](https://keda.sh/docs/latest/deploy/).
 install the broker-controller from this repository:
 
 ```
-ko apply -f config/broker/
+kubectl create namespace knative-eventing
```

Comment:
This should have been created for you as per L7 (installing eventing creates the knative-eventing namespace).